packages/cli/src/commands/metadata/command.ts

```diff
@@ -208,10 +183,39 @@ export function updatePackageMetadata(
   packageJSON.homepage = new URL(opts.homepage);
   packageJSON.bugs = new URL(opts.bugs);
 
-  // add keywords (replace some), then uniquify
-  addNewKeywords(packageJSON, opts);
+  // initialize empty string array if not already present
+  if (!packageJSON.keywords) {
+    packageJSON.keywords = [];
+  }
+
+  // if already have keywords, replace lifecycle and support with new values (if defined)
```

Comment:
does sonarlint not complain about this method being too complex?

Reply:
Nope